Remotion/Data/DomainObjects/Persistence/Rdbms/SqlServer/Sql2016/SqlStorageObjectFactory.cs

```diff
@@ -186,7 +186,7 @@ public virtual IRdbmsPersistenceModelProvider CreateRdbmsPersistenceModelProvide
       var storageTypeInformationProvider = CreateStorageTypeInformationProvider(storageProviderDefinition);
       var storageNameProvider = CreateStorageNameProvider(storageProviderDefinition);
       var persistenceModelProvider = CreateRdbmsPersistenceModelProvider(storageProviderDefinition);
-      var storageProviderDefinitionFinder = new StorageEntityBasedStorageProviderDefinitionFinder();
+      var storageProviderDefinitionFinder = SafeServiceLocator.Current.GetInstance<IStorageSettings>();
```

Comment:
I am not sure if we want to replace the `StorageEntityBasedStorageProviderDefinitionFinder` as well as it has nothing to do with the config but rather uses the class definition to get to the storage provider definition. This is something we'll have to check with @MichaelKetting

Reply:
Yes, the idea is to clean/unify this api. Small stuff that's related to the rewrite should be included to have a clean new API.